libs/shared/tests/scenarios/complete-scenario.builder.ts

```diff
@@ -216,6 +217,12 @@ export class CompleteScenarioBuilder {
         .addRole(ServiceRoleEnum.INNOVATOR, 'innovatorRole')
         .save();
 
+      // Collaborator on johnInnovationArchived
+      const janeInnovatorArchived = await new UserBuilder(entityManager)
+        .setName('Jane Innovator Archived')
+        .addRole(ServiceRoleEnum.INNOVATOR, 'innovatorRole')
+        .save();
```

Comment:
I think it's ok to use the same Jane as we have as the collaborator in the archived innovation. Or would mess with other things?